code/about.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Textildesign</title>
+  <link rel="stylesheet" href="./style.css" />
+  <!-- dont forget the meta tag for responsiveness -->
+</head>
+
+<body>
+  <header>
+    <div class="hamburger">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+    <div class="header-text">
+      <h1>Anna Leutert</h1>
+      <h2>Handweberin, Textil- und Farbdesignerin</h2>
```

Comment:
This is also very good. Using a correct hierarchy in titles/text-tags makes your site better according to Google. Meaning your SEO is on point! 🙂